src/main/java/org/jvnet/hudson/plugins/m2release/M2ReleaseAction.java

```diff
@@ -71,8 +72,8 @@
 	private boolean selectCustomScmTag = false;
 	private boolean selectAppendHudsonUsername;
 	private boolean selectScmCredentials;
-
-	public M2ReleaseAction(MavenModuleSet project, boolean selectCustomScmCommentPrefix, boolean selectAppendHudsonUsername, boolean selectScmCredentials) {
+	private boolean enableProduct = false;
```

Comment:
It would be better to call this isProduct, as we use this as a flag to mark particular job a product build or not